README.md

```diff
@@ -565,6 +566,7 @@ Follows best practices and conventions to provide you a SOLID development experi
 * [Scrutor](https://github.com/khellang/Scrutor) - Assembly scanning extensions for Microsoft.Extensions.DependencyInjection.
 * [Sheller](https://github.com/twitchax/Sheller) - A .NET library that makes shelling out commands super easy and fluent.
 * [SmartFormat.NET](https://github.com/scottrippey/SmartFormat.NET) - An extensible replacement for string.Format.
+* [Spectre.Console](https://github.com/spectreconsole/spectre.console) - A .NET 5/.NET Standard 2.0 library that makes it easier to create beautiful, cross platform, console applications.
```

Comment:
Why do we need this ?

Reply:
Because its awesome?

But seriously. It looks like it well made, has a lot of unit test, and a dedicated site for documentation describing every aspect of the library.

Reply:
@ngohungphuc Why did we not merge this? What is the opposition?